README.md

```diff
@@ -95,6 +95,10 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
 ```sh
 wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
 ```
+```sh
+NVM_LATEST_VERSION=$(curl --silent "https://api.github.com/repos/nvm-sh/nvm/releases/latest" | rg -e '"tag_name":\s"(.*)".*' -r '$1')
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/${NVM_LATEST_VERSION}/install.sh | bash
+```
```

Comment:
i've very intentionally hardcoded the version number in the install command; i *want* users to come to this document to get the updated install command, so they're not blindly receiving breaking changes.

Reply:
As I begin to use NVM in my CI pipeline, I have to make a choice whether to use hard-coded version or not.
And I just find out the section [Manual Install](https://github.com/nvm-sh/nvm#manual-install) in [README](https://github.com/nvm-sh/nvm#readme), which is not hardcoded, and is what I want at first.

Appreciate for the explanation!

Reply:
It is a very bad idea to do that - as I, the only maintainer and thus the only authority on this subject in the universe, am saying, you should always depend on a hardcoded version, and simply accept that you'll need to sign up to keep this up to date manually.